### PR TITLE
Test/16/write unit test for player class

### PR DESCRIPTION
**Pull request by kernohad**

### CODE REVIEW

File: src/Model/GamePackage/Game.java [hunk 1]
```diff
@@ -1,4 +1,557 @@
...
+import java.util.stream.Stream;
+
+/**********************************************************************
+ * The Game class is responsible for almost all of the game logic.
+ * Any action that goes on in the game is handled here.
+ *
+ * @author Dylan Kernohan
+ * @version 2/19/2018
+ *********************************************************************/
 public class Game {
```
Comment (Dcbaas) [inline src/Model/GamePackage/Game.java]: Is there going to be a separate pull request for the game class specifically or should I review it here in this pull request?

Comment (kernohad) [reply src/Model/GamePackage/Game.java]: Crap. Yeah that was not supposed to be in this. I will try and remove it

**Comment by kernohad** [comment]
I deleted the Game and MainTest files. Go ahead and review the rest

File: src/Model/BoardPackage/Board.java [hunk 2]
```diff
@@ -154,5 +154,11 @@ public void sendToJail(Player player) {
         return squaresList.stream().map(squaresList -> (OwnableSquare) squaresList)
                 .filter(ownableSquare -> ownableSquare.getGROUP_NUMBER() == groupId).collect(Collectors.toList());
     }
+
+    public ArrayList<BoardSquare> getLocationByType(int typeID) {
```
Comment (quirogas) [inline src/Model/BoardPackage/Board.java]: you also included some other changes that I made while working on the game class. You should get rid of this too.

File: src/Model/CardPackage/Deck.java [hunk 3]
```diff
@@ -22,20 +22,22 @@
      *  of cards.
      *
      * @param fileName
-     * @param cardType
+     * @param cardType True for Chance, False for Community Chest
      *********************************************************************************/
     public Deck(String fileName, boolean cardType) {
+        //intializes the deck
+        deck = new ArrayList<>();
```
Comment (quirogas) [inline src/Model/CardPackage/Deck.java]: this too

File: src/Model/CardPackage/Deck.java [hunk 4]
```diff
@@ -67,4 +69,20 @@ public boolean returnCard(Card card){
     public void shufleDeck(){
         Collections.shuffle(deck);
     }
+
+    /**
+     *
+     * @return
+     */
```
Comment (quirogas) [inline src/Model/CardPackage/Deck.java]: this too

File: src/Model/GamePackage/Player.java [hunk 5]
```diff
@@ -345,4 +345,10 @@ public boolean addGroupOwned(int groupNumber){
     public boolean removeGroupOwned(int groupNumber){
         return groupsOwned.remove((Integer) groupNumber);
     }
+
+	public void recieveCard(Card card) {
+	}
```
Comment (quirogas) [inline src/Model/GamePackage/Player.java]: this too

File: src/mainTest.java [hunk 6]
```diff
@@ -1,21 +0,0 @@
...
- */
-public class mainTest {
-
-    /*********************************************************************************************
-     * Main method
-     * @param args
-     ********************************************************************************************/
-    public static void main(String[] args) {
-        Board board = new Board(System.getenv("FILENAME"));
-        board.getSquaresList().forEach(property -> System.out.println(property.getName()));
```
Comment (quirogas) [inline src/mainTest.java]: Did you deleted this class ?
